webui/react/src/pages/F_ExpList/ComparisonView.tsx

```diff
@@ -94,10 +95,9 @@ const ComparisonView: React.FC<Props> = ({
 
   const rightPane =
     selectedExperiments.length === 0 ? (
-      <Message
+      <Alert
         description="Select experiments you would like to compare."
-        icon="warning"
-        title="No experiments selected."
+        message="No experiments selected."
```

Comment:
On other sections `icon="warning"` is replaced with `type="warning"` and would be added here for a 1:1 replacement

This is a large panel which I'd be OK to contain a Message or to change the intensity of the message. Maybe check in on Slack, we have been discussing how some experiments may be selected on an unloaded section so the wording is misleading.

Reply:
I think it used to be an `Alert` without a type, and I just added `icon="warning"` when I changed it to a `Message`.

If you think Message with warning icon is most appropriate for this, I can go ahead and revert the changes for this file.

Reply:
ok that makes sense if it's going back to the previous component + props